src/app/views.py

```diff
@@ -127,7 +127,7 @@ def submitNewLicense(request):
                         # This is present only when executing submit license via tests
                         urlType = request.POST["urlType"]
 
-                    matchingIds, matchingType = utils.check_spdx_license(licenseText)
+                    matchingIds, matchingType, _ = utils.check_spdx_license(licenseText)
```

Comment:
Should we show all matches with percent match on UI as well? Maybe let's open a separate task for that.

Reply:
Probably, but yes, this is another task

Reply:
Agree as well - looks like some good improvements in this PR